go/server/templates/microbench.tmpl

```diff
@@ -35,20 +36,20 @@
                     <tr>
                         <td>{{ $val.PkgName }}</td>
                         <td><a href="https://github.com/vitessio/vitess/search?q={{ $val.Name }}&type=code" target="_blank">{{ $val.Name }}</a></td>
-                        <td class="text-right">{{ $val.Current.Ops }}</td>
-                        <td class="text-right">{{ $val.Last.Ops }}</td>
+                        <td class="text-right">{{ $val.Current.OpsStr }}</td>
+                        <td class="text-right">{{ $val.Last.OpsStr }}</td>
+                        <td class="text-right">{{ $val.CurrLastDiffStr }}</td>
+                        <td class="text-right {{if le $val.CurrLastDiff 0.90 }} bg-success {{ else if ge $val.CurrLastDiff 1.10 }} bg-danger {{ end }}">{{ $val.Current.NSPerOpStr }}</td>
```

Comment:
Now that we have a clear rendered % difference, should we move the green/red indicator to put it on the `{{ $val.CurrLastDiffStr }}` column instead of the `NSPerOps` one?

Reply:
I like